### ISSUE TITLE
inline optimization is leaving behind tracking markers for inline `<script>` tags

### ISSUE THREAD

**Issue by thescientist13** [ProjectEvergreen/greenwood#784]
## Type of Change
- [ ] New Feature Request
- [ ] Documentation / Website
- [ ] Improvement / Suggestion
- [x] Bug
- [ ] Other (please clarify below)

## Summary
Noticed in [one of our specs](https://github.com/ProjectEvergreen/greenwood/blob/v0.18.0/packages/cli/test/cases/build.config.optimization-inline/build.config-optimization-inline.spec.js#L123) that that the `console.log` marker we use when tracking inline `<script>` tags was not getting properly cleaned up.

<img width="740" alt="greenwood-rollup-marker-in-sourcecode" src="https://user-images.githubusercontent.com/895923/139597194-b67ed304-8ece-47a6-b82c-4ad8f9309f4e.png">

## Details
Basically we had something like this in our test
```html
<script type="module">
  class Bar {
  }
  
  export Bar;
</script>
```

And the output in the HTML was (minified of course)
```html
<script type="module">
  class Bar {
  }
  
  export Bar;
  
  console.log('xxxxx-scratch-xxxxx');
</script>
```

We definitely don't want that `console` in there 😨
